angular/src/lib/modal/modal.component.ts

```diff
@@ -39,7 +39,11 @@ export type SizeType =
   .md-modal.md-modal--full {
       position: static;
     }
-  `]
+  `, `
+  ::ng-deep .md-modal__backdrop {
+      z-index: 1000;
+    }
+  `],
```

Comment:
this really doesn't seem like what we should do.

Reply:
In core, we included https://github.com/momentum-design/momentum-ui/tree/master/core/scss/components/cdk-overlay and adjusted the z-index .. so if you are importing the cdk.css you should not.

Reply:
Cool thanks, will use those styles and close this PR as it is a non-issue.